hotness/buildsys.py

```diff
@@ -60,7 +62,14 @@ def __init__(self, consumer, config):
 
     def session_maker(self):
```

Comment:
If you are doing multiple threads at the same time, you want to use a lock in this function, since having multiple threads be in the middle of krb auth at the same time is problematic.